cmd/triggerRun/cmd/root.go

```diff
@@ -43,12 +54,51 @@ var (
 func init() {
 	rootCmd.Flags().StringVarP(&triggerFile, "triggerFile", "t", "", "Path to trigger yaml file")
 	rootCmd.Flags().StringVarP(&httpPath, "httpPath", "r", "", "Path to body event")
+	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "absolute path to the kubeconfig file")
 }
 
 func rootRun(cmd *cobra.Command, args []string) {
 	// TODO: Not implemented
 }
 
+func trigger(w io.Writer, triggerFile, httpPath string) error {
+	// Read HTTP request.
+	r, err := readHTTP(httpPath)
+	if err != nil {
+		return fmt.Errorf("error reading HTTP file: %w", err)
+	}
+	defer r.Body.Close()
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		return fmt.Errorf("error reading HTTP body: %w", err)
+	}
+
+	// Read triggers.
+	triggers, err := readTrigger(triggerFile)
+	if err != nil {
+		return fmt.Errorf("error reading triggers: %w", err)
+	}
+
+	client, err := GetKubeClient("kubeClient")
```

Comment:
I don't see usage of `GetKubeClient` outside the package so we can make it lowercase 
```suggestion
	client, err := getKubeClient("kubeClient")
```